ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/chooseserver/AvailableServerProvider.kt

```diff
@@ -0,0 +1,39 @@
+package co.electriccoin.zcash.ui.screen.chooseserver
+
+import cash.z.ecc.android.sdk.model.ZcashNetwork
+import cash.z.ecc.sdk.extension.Mainnet
+import cash.z.ecc.sdk.extension.Testnet
+import co.electriccoin.lightwallet.client.model.LightWalletEndpoint
+import kotlinx.collections.immutable.toImmutableList
+
+// TODO [#1273]: Add ChooseServer Tests #1273
+// TODO [#1273]: https://github.com/Electric-Coin-Company/zashi-android/issues/1273
+
+object AvailableServerProvider {
+    // North America  | [na.lightwalletd.com](http://na.lightwalletd.com/) | 443
+    // South America | [sa.lightwalletd.com](http://sa.lightwalletd.com/) | 443
+    // Europe & Africa | [eu.lightwalletd.com](http://eu.lightwalletd.com/) | 443
+    // Asia & Oceania | [ai.lightwalletd.com](http://ai.lightwalletd.com/) | 443
+    // Plus current network defaults:
+    // Mainnet: mainnet.lightwalletd.com | 9067
+    // Testnet: lightwalletd.testnet.electriccoin.co | 9067
```

Comment:
[non-blocker] Are these comments needed? Everything can be easily determined from the code bellow it.